module-system/sov-state/src/zk_storage.rs

```diff
@@ -91,4 +91,9 @@ impl<S: MerkleProofSpec> Storage for ZkStorage<S> {
 
         Ok(new_root.0)
     }
+
+    // ZK Storage is never empty
+    fn is_empty(&self) -> bool {
+        false
```

Comment:
This could cause a divergence between native and zk storage, which could lead to very unintuitive bugs. 
 I think the best thing to do is to check if its state_root is equal to the root of an empty JMT, but we can also leave it `unimplemented!()` for now.